x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_ecommerce/ml/high_sum_total_sales.json

```diff
@@ -0,0 +1,39 @@
+{
+  "groups": ["kibana_sample_data"],
+  "description": "Kibana sample eCommerce data : detect anomalies in total sales",
+  "analysis_config" : {
+    "bucket_span": "1h",
+    "detectors": [
+      {
+        "detector_description": "Mean total sales",
+        "function": "high_sum",
+	"field_name": "taxful_total_price",
+	"over_field_name": "customer_full_name.keyword"
+      }
+    ],
+    "influencers": ["customer_full_name.keyword", "category.keyword"]
+  },
+  "analysis_limits": {
+    "model_memory_limit": "10mb"
+  },
+  "data_description": {
+    "time_field": "order_date"
+  },
+  "model_plot_config": {
+    "enabled": true
+  },
+  "custom_settings": {
+    "created_by": "ml-module-sample",
+    "custom_urls": [
+      {
+        "url_name": "Raw data",
+        "url_value": "kibana#/discover?_g=(time:(from:\u0027$earliest$\u0027,mode:absolute,to:\u0027$latest$\u0027))&_a=(index:ff959d40-b880-11e8-a6d9-e546fe2bba5f)"
+      },	
+      {
+        "url_name": "Data dashboard",
+        "url_value": "kibana#/dashboard/722b74f0-b882-11e8-a6d9-e546fe2bba5f?_g=(filters:!(),time:(from:\u0027$earliest$\u0027,mode:absolute,to:\u0027$latest$\u0027))&_a=(filters:!((\u0027$state\u0027:(store:appState),meta:(alias:!n,disabled:!f,index:\u0027INDEX_PATTERN_ID\u0027,key:category.keyword,negate:!f,params:(query:\u0027$category.keyword$\u0027),type:phrase,value:\u0027$category.keyword$\u0027),query:(match:(category.keyword:(query:\u0027$category.keyword$\u0027,type:phrase))))),query:(language:kuery,query:\u0027\u0027))"
```

Comment:
The `over_field` `customer_full_name.keyword` should be used in the custom URL rather than the influencer `category.keyword`, which may not be present in the record (and this would also avoid hitting the issue with custom URLs where the field used in the URL has multiple values in the record).